server/data/series.js

```diff
@@ -1,5 +1,17 @@
 import {List} from 'immutable';
 
+export function getPositionInSeries(tags: {}): ?number {
+  const chapterString = 'chapter';
+  const chapterTag = Object.keys(tags).find((tag) => tag.toLowerCase().startsWith(chapterString));
```

Comment:
This is on the server so I don't think we have to care do we? (i.e. it's in Chrome)